src/mutate.rs

```diff
@@ -27,6 +27,7 @@ pub enum Genre {
     FnValue,
     /// Replace `==` with `!=` and so on.
     BinaryOperator,
+    UnaryOperator,
```

Comment:
Please also describe this pattern in https://github.com/sourcefrog/cargo-mutants/blob/main/book/src/mutants.md